react/features/chat/middleware.ts

```diff
@@ -477,7 +487,7 @@ function _handleReceivedMessage({ dispatch, getState }: IStore,
 
         APP.API.notifyReceivedChatMessage({
             body: message,
-            id,
+            participantId,
```

Comment:
@he-patrick This is what breaks the external API test. You need to adjust this: https://github.com/jitsi/jitsi-meet/blob/b2e06c1c94803144735c8b8f91e13bd01bc48503/modules/API/API.js#L1348

Please keep "from" there or applications will break.